### ISSUE TITLE
Drop/Hover target with childrens

### ISSUE THREAD

**Issue by daugsbi** [yahoo/react-dnd-touch-backend#71]
### Issue
I am experiencing the following issue since the version >= 0.3.7:

If the drop target has nested objects and the hover happens over such a nested object, i. e. a svg group containing a circle and a text, the corresponding target is filtered by this statement: 

```javascript
let orderedDragOverTargetIds = elementsAtPoint 
  // Filter off nodes that arent a hovered DropTargets nodes
  .filter(node => dragOverTargetNodes.indexOf(node) > -1)
```
_Because the node is not an element of dragOverTargetNodes_.

### Expected behavior
Not filtered out, because the parent is a dragOverTarget and a legitimate drop target. 

### Recommendation
Check parents until a dragOverTargetNode is identified.

I could create a pull request but like to discuss the implications first.

**Comment by JoaoMosmann** [comment]
The ordering was implemented so it would have the same behavior as the html5-backend.

Could you reproduce the problem in a jsfiddle?

There is a demo in the repo that shows nested drop targets working fine.
Run `yarn run dev` and navigate to `http://localhost:7789/dropTarget/` open the devtools to see the drop results.

**Comment by daugsbi** [comment]
I have reproduced the problem with the example in the following commit:
daugsbi@ef1edcd

I have tested chrome and firefox. Both are not working with the group as a drop target. It is not returned by elementsatpoint, so it can not match.

Could be an edge case, but would be nice to also drop on svg groups ;)

**Comment by JoaoMosmann** [comment]
Indeed, it is not working with the SVG. It must have something to do with some SVG point-events limitation.

https://github.com/yahoo/react-dnd-touch-backend/blob/master/src/Touch.js#L27
As you can see, in the `elementsFromPoint` polyfill, we use pointerEvents to work around the layers of elements. But as SVG elements don't allow `pointer-events` it doesn't work. The native `elementsFromPoint` use the same approach and don't work either.

A PR is very welcomed. Otherwise, I'll try to fix it this weekend.

Thanks for reporting it.
